site/docs/v1/tech/migration-guide/_what-next.adoc

```diff
@@ -1,22 +1,22 @@
-You now have your users migrated. Congratulations! What is next?
+You now have your users migrated, or a plan to do so. Congratulations! What is next?
 
 You need to migrate the other configuration, as mentioned in <<Other Entities>>. Since the type of configuration varies, it is hard to provide a full list of how to import these items, but the general pattern will be:
```

Comment:
```suggestion
You need to migrate additional configuration, as mentioned in <<Other Entities>>. Since the type of configuration varies, it is hard to provide a full list of how to import these items, but the general pattern will be:
```